opsdroid/connector/matrix/create_events.py

```diff
@@ -47,10 +47,10 @@ def __init__(self, connector):
     async def create_message(self, event, roomid):
         """Send a Message event."""
         return events.Message(
-            event["content"]["body"],
-            await self.connector.get_nick(roomid, event["sender"]),
-            roomid,
-            self.connector,
+            text=event["content"]["body"],
+            user=await self.connector.get_nick(roomid, event["sender"]),
+            target=roomid,
+            connector=self.connector,
```

Comment:
we are going to want a `user_id=event['sender']` here, and in all the other events in this file.